Patch provided from upstream
Add dependencies for Lua libraries to Makefile so that it can be build with multiple make jobs
https://github.com/rrze-likwid/likwid/commit/bf12653e199d6d60055982bd92b284f44197bccf

--- ext/lua/Makefile.org	2015-08-21 09:53:48.430859982 +0800
+++ ext/lua/Makefile	2015-08-21 09:51:50.905236819 +0800
@@ -42,10 +42,10 @@
 $(BUILD_DIR):
 	@mkdir $(BUILD_DIR)
 
-$(STATIC_LIBLUA):
+$(STATIC_LIBLUA): $(OBJ)
 	$(Q)${AR} -cq $(LIBLUA) $(OBJ)
 
-$(SHARED_LIBLUA):
+$(SHARED_LIBLUA): $(OBJ)
 	$(Q)$(CC) $(LFLAGS) -Wl,-soname,$(LIBLUA).$(VERSION) -shared -fPIC -o $(LIBLUA) $(OBJ) $(LIBS)
 
 $(INTERPRETER): $(SHARED_LIBLUA) $(BUILD_DIR)/lua.o
